client/client.go

```diff
@@ -129,6 +136,21 @@ func buildKubeClient(kubeConfig api.Config, ctx string) (*kubernetes.Clientset,
 	return kubernetes.NewForConfig(restConfig)
 }
 
+func getApisMap(client *kubernetes.Clientset) (map[string]struct{}, error) {
```

Comment:
```suggestion
func getAPIsMap(client *kubernetes.Clientset) (map[string]struct{}, error) {
```

Total nit-pick, but the convention in Go is to make abbreviations like `API` and `ID` uppercase when it doesn't cause the variable to be made public. [Reference](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)